types/types.go

```diff
@@ -139,7 +139,8 @@ type NodeConfig struct {
 	// status that is set by containerlab to indicate deployment stage
 
 	// Extras
-	Extras *Extras `json:"extras,omitempty"` // Extra node parameters
+	Extras  *Extras  `json:"extras,omitempty"` // Extra node parameters
+	WaitFor []string `json:"waitFor,omitempty"`
```

Comment:
json and yaml tags for composite words should use dashes
`waitFor` -> `wait-for`

This is aligned with the rest of the fields we expose in clab config

Reply:
done!